src/symbols.c

```diff
@@ -2289,6 +2290,32 @@ static gint get_fold_header_after(ScintillaObject *sci, gint line)
 }
 
 
+/* returns the line after following all brace match for @brace on @line */
+static gint resolve_matching_braces(ScintillaObject *sci, gint line, gint brace)
+{
+	gint pos = sci_get_position_from_line(sci, line);
+	gint line_end = sci_get_line_end_position(sci, line);
+	gint lexer = sci_get_lexer(sci);
+
+	for (; pos < line_end; pos++) {
```

Comment:
> but yes, I do think there is such thing as over-splitting, too (basically when it ends up in split things that don't have a defined goal by themselves).

Its always a judgement call, and it depends critically on the name of the function being obvious, otherwise everybody has to go "oh, `resolve_brace_match` hmmm, what does that do? what does "resolve" mean in this context? I guess I better go find its definition and read it.  Which is a waste of everybodys time instead of having a few extra lines in the function, which you can even comment :)